lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -95,8 +101,11 @@
   private final int serverPort;
   private final SocketAddress localAddress;
 
-  private final NettyClientEventListener clientEventListener;
-  private final NettyServerEventListener serverEventListener;
+  private final AbstractNettyEventListener clientEventListener;
+  private final AbstractNettyEventListener serverEventListener;
+
+  private final boolean ssl = System.getProperty("ssl") != null;
```

Comment:
I don't think initializing `final` variable here is good, because it is not a static variable. How about moving this into the constructor?